### ISSUE TITLE
Broken after update

### ISSUE THREAD

**Issue by sodbileg** [Templarian/MaterialDesign#492]
I updated to v1.1.70. Most of the icons are broken or changed after that. Does icon name and content get changed on updates? For example, `content: "\f1ca"` was checkbox-marked-circle, but it is calendar-remove now.

**Comment by Templarian** [comment]
Icon names rarely change between releases. The character positions change between every release. Please use SASS to compile your CSS.

```sass
content: char(nth($hexes, index($names, 'checkbox-marked-circle')))
```

If you do not plan to use SASS there is no other way than manually updating each of the character's after every webfont update. For Photoshop/Illustrator users a font with fixed hex codes will be created #247.

**Comment by sodbileg** [comment]
Is there a LESS way to do that?

**Comment by Templarian** [comment]
I keep saying I'm going to make a LESS file, but things keep distracting me.

I'm always open to a pull request for issue #150. _Eventually_ I will add LESS support. I just don't have a timeline right now. All my focus is on this new export screen at the moment.

**Comment by Templarian** [comment]
This was resolved in the v1.5.54 release if you are not following the main thread.
